src/webgl/buffer.js

```diff
@@ -120,7 +83,7 @@ export default class Buffer {
   setData({
     data,
     bytes,
-    target = GL.ARRAY_BUFFER,
+    target = this.target || GL.ARRAY_BUFFER,
```

Comment:
> Is this.target still available for the Buffer class?

In my latest test I was surprised to see that I couldn't change the target of a Buffer once created. So setting the target is important.

Is that the answer or is it the use of `this.target` as a default parameter that is the concern? It works well, although I can't find a clear guarantee for it in the standard.